packages/twenty-server/src/engine/core-modules/application/application.service.ts

```diff
@@ -32,6 +58,8 @@ export class ApplicationService {
         'applicationVariables',
       ],
     });
+
+    return this.decryptManyApplicationVariables(applications);
```

Comment:
I don't like the naming: decryptManyApplicationVariables should take variables as parameter. Here it's more decryptVariablesWithinApplication